docs/usage.md

```diff
@@ -222,54 +304,54 @@ Barcode kit used to perform the sequencing e.g. `SQK-PBK004`. When `--skip_basec
 
 ### `--guppy_config`
 
-Guppy config file used for basecalling passed with the `--config` parameter. Cannot be used in conjunction with `--flowcell` and `--kit`.
-This can be a local file (i.e. `/your/dir/guppy_conf.cfg`) or a string specifying a configuration stored in the `/opt/ont/guppy/data` directory of Guppy.
+Config file used for basecalling that will be passed to Guppy via the "--config" parameter. Cannot be used in conjunction with `--flowcell` and `--kit`.
+This can be a local file (i.e. `/your/dir/guppy_conf.cfg`) or a string specifying a configuration stored in the `/opt/ont/guppy/data/` directory of Guppy.
 
 ### `--guppy_model`
 
-Custom basecalling model file (`json`) to pass to Guppy for basecalling with the `--model` parameter. Custom basecalling models can be trained with software such as [Taiyaki](https://github.com/nanoporetech/taiyaki). This can also be a string specifying a model stored in the `/opt/ont/guppy/data` directory of Guppy.
+Custom basecalling model file in `json` format that will be passed to Guppy via the "--model" parameter. Custom basecalling models can be trained with software such as [Taiyaki](https://github.com/nanoporetech/taiyaki). This can also be a string specifying a model stored in the `/opt/ont/guppy/data` directory of Guppy.
 
 ### `--guppy_gpu`
 
-Whether to demultiplex with Guppy in GPU mode.
+Whether to demultiplex with Guppy in GPU mode (default: false).
 
 ### `--guppy_gpu_runners`
 
-Number of '--gpu_runners_per_device' used for guppy when using `--guppy_gpu` (default: 6)
+Number of "--gpu_runners_per_device" used for Guppy when using `--guppy_gpu` (default: 6).
 
 ### `--guppy_cpu_threads`
 
-Number of '--cpu_threads_per_caller' used for guppy when using `--guppy_gpu` (default: 1)
+Number of "--cpu_threads_per_caller" used for Guppy when using `--guppy_gpu` (default: 1).
 
 ### `--gpu_device`
 
-Basecalling device specified to Guppy in GPU mode using `--device` (default: 'auto')
+Basecalling device specified to Guppy in GPU mode using "--device" (default: 'auto').
```

Comment:
why is the formatting on "--device" not `--device` like the rest of the parameters shown above ?

Reply:
nevermind, realized what you were doing, smart!  (not a part of parameters for the pipeline but for gpu)